src/Symfony/Component/Notifier/Bridge/ContactEveryone/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+CHANGELOG
+=========
+
+6.1
```

Comment:
This will go in 6.2, not in 6.1. The 6.1 branch is already in feature freeze since 1.5 month